python/script/ci-test

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
-bundle install
+pyenv exec pip --disable-pip-version-check install --use-pep517 -r "dev-requirements.txt"
```

Comment:
Could we simply do:
```shell
pyenv exec pip --disable-pip-version-check install --use-pep517 flake8==5.0.4
```

That'd remove the need for `dev-requirements.txt` unless/until we start adding additional packages there...

Reply:
Yep, I also though of that, but then that dependency won't be watched but Dependabot and will fall out of date.

Reply:
good thought, probably better to keep it in a `*requirements.txt` file then...